src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,13 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User user = this.userService.findByEmail(email);
+        if (user == null) {
+            return false;
+        }
+        return user.getPassword().equals(password);
```

Comment:
Just a hint) We can combine these checks together, but your solution can be more flexible for logging/debugging
```suggestion
        return user != null && user.getPassword().equals(password);
```